Src/traceDecoder_etm4.c

```diff
@@ -404,6 +409,8 @@ static bool _pumpAction( struct TRACEDecoderEngine *e, struct TRACECPUState *cpu
                         cpu->addr = j->q[match].addr;
                         retVal = TRACE_EV_MSG_RXED;
                         _stateChange( cpu, EV_CH_ADDRESS );
+                        _stackQ( j );
```

Comment:
After Exact Match Address the function update_address_regs is called therefore a stackQ is needed